examples/src/main/java/org/bitcoinj/examples/ExamplePaymentChannelClient.java

```diff
@@ -54,8 +56,22 @@
 
     public static void main(String[] args) throws Exception {
         BriefLogFormatter.init();
-        System.out.println("USAGE: host");
-        new ExamplePaymentChannelClient().run(args[0]);
+        System.out.println("USAGE: [--net={MAIN/TEST/REGTEST}] host");
```

Comment:
In several tools, we're using "jopt" for argument parsing. Perhaps you want to introduce it here as well?